base/static/base/js/jquery.track-everything.js

```diff
@@ -63,362 +29,533 @@
+                        let currentOption = searchButton.getAttribute('data-ga-event-option') || '';
+                        if (target.checked) {
+                            // Add option
+                            if (currentOption) {
+                                if (!currentOption.split('; ').includes(optionText)) {
+                                    currentOption += '; ' + optionText;
+                                }
+                            } else {
+                                currentOption = optionText;
+                            }
+                        } else {
+                            // Remove option
+                            currentOption = currentOption.split('; ').filter(opt => opt !== optionText).join('; ');
+                        }
+                        searchButton.setAttribute('data-ga-event-option', currentOption);
+                    });
+                }
+            },
+            radio(target, searchWidget) {
+                const searchButtons = searchWidget.querySelectorAll('button.btn-search[type="submit"][data-ga-subcategory="' + target.getAttribute('data-ga-subcategory') + '"]');
+                if (searchButtons.length) {
+                    searchButtons.forEach((searchButton) => {
+                        let optionText = target.getAttribute('data-ga-label');
+                        let currentOption = searchButton.getAttribute('data-ga-event-option') || '';
+                        let optionsArr = currentOption ? currentOption.split('; ') : [];
+                        // Remove any previous rad:... entry
+                        optionsArr = optionsArr.filter(opt => !opt.startsWith('radio:'));
+                        if (target.checked) {
+                            optionsArr.push('radio:' + optionText);
+                        }
+                        currentOption = optionsArr.join('; ');
+                        searchButton.setAttribute('data-ga-event-option', currentOption);
+                    });
+                }
+            },
+            select(target, searchWidget) {
+                const searchButton = searchWidget.querySelector('button[type="submit"].btn-search');
+                if (searchButton) {
+                    let selectedValue = target.value.trim();
+                    let currentOption = searchButton.getAttribute('data-ga-event-option') || '';
+                    let optionsArr = currentOption ? currentOption.split('; ') : [];
+                    // Remove any previous type:... entry
+                    optionsArr = optionsArr.filter(opt => !opt.startsWith('type:'));
+                    if (selectedValue !== 'AllFields') {
+                        optionsArr.push('type:' + selectedValue);
+                    }
+                    currentOption = optionsArr.join('; ');
+                    searchButton.setAttribute('data-ga-event-option', currentOption);
+                }
+            }

```

Comment:
Good example of `ES6` syntax that I mentioned in the other comment. Since we're doing this in parts of the code, we should just do it everywhere and not use `var`.